examples/integration/BUILD

```diff
@@ -6,6 +6,10 @@ load(":xcodeproj_targets.bzl", "CONFIG", "XCODEPROJ_TARGETS")
 
 xcodeproj(
     name = "xcodeproj",
+    associated_extra_files = {
+        "//Lib": ["//Lib:README.md"],
+        "//iOSApp/Source:iOSApp": ["//iOSApp:ownership.yaml"],
```

Comment:
🔡 
```suggestion
        "//iOSApp/Source:iOSApp": ["//iOSApp:ownership.yaml"],
        "//Lib": ["//Lib:README.md"],
```

Reply:
(Though, if `buildifier` made it this order, Because Capital < lowercase, then :+1:.)

Reply:
`buildifier` doesn't mind this it seems, so I changed to alphabetical.